ingest/clawback_details.go

```diff
@@ -0,0 +1,48 @@
+package ingest
+
+import "fmt"
+
+type ClawbackDetail struct {
+	AssetCode   string `json:"asset_code"`
+	AssetIssuer string `json:"asset_issuer"`
+	AssetType   string `json:"asset_type"`
+	From        string `json:"from"`
+	FromMuxed   string `json:"from_muxed"`
+	FromMuxedID uint64 `json:"from_muxed_id"`
+	Amount      int64  `json:"amount"`
```

Comment:
any and all amounts that exceed `int32` need to be encoded as strings in the json
```suggestion
	Amount      int64  `json:"amount,string"`
```

Reply:
Updated in cc6d49df2de93fbffceb5557a714a3166d38de82